packages/components/src/vue/vuex-modules/data/lexis.js

```diff
@@ -121,13 +121,14 @@ export default class Lexis extends Module {
   }
 
   static async getTreebankWordIds (treebankDataItem, textSelector) {
+    const lm = LMF.getLanguageModel(textSelector.languageID)
     const findWordResult = await ClientAdapters.morphology.arethusaTreebank({
       method: 'findWord',
       params: {
         provider: treebankDataItem.provider,
         word: textSelector.normalizedText,
-        prefix: textSelector.textQuoteSelector.prefix,
-        suffix: textSelector.textQuoteSelector.suffix,
+        prefix: lm.normalizeWord(textSelector.textQuoteSelector.prefix),
```

Comment:
did you decide not to rename the function?

Reply:
Oh, that slipped my mind among other changes. Will rename! Thanks for catching this!